composer.json

```diff
@@ -21,14 +21,14 @@
         "ext-curl":"*",
         "phpcr/phpcr": "~2.1.2",
         "phpcr/phpcr-utils": "1.*,>=1.1.2",
-        "jackalope/jackalope": "~1.2.0"
+        "jackalope/jackalope": "dev-node-get-definition as 1.2.1"
     },
     "provide": {
         "jackalope/jackalope-transport": "1.1.0"
     },
     "require-dev": {
         "psr/log": "~1.0",
-        "phpcr/phpcr-api-tests": "2.1.4",
+        "phpcr/phpcr-api-tests": "dev-node-get-definition as 2.1.5",
```

Comment:
adjust once https://github.com/phpcr/phpcr-api-tests/pull/163 is merged and possibly others, and we were able to tag a release